plugin/builder-lxc/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"github.com/mitchellh/packer/builder/lxc"
+	"github.com/mitchellh/packer/packer/plugin"
+)
+
+func main() {
+	server, err := plugin.Server()
+	if err != nil {
+		panic(err)
+	}
+	server.RegisterBuilder(new(lxc.Builder))
+	server.Serve()
+}
```

Comment:
This file is not needed anymore.

Reply:
@SwampDragons This was not fixed, can you remove this and retest?

Reply:
oops!  https://github.com/hashicorp/packer/pull/5402